server/src/main/java/org/opensearch/OpenSearchException.java

```diff
@@ -1594,6 +1594,15 @@ private enum OpenSearchExceptionHandle {
             org.opensearch.transport.NoSeedNodeLeftException::new,
             160,
             LegacyESVersion.V_7_10_0
+        ),
+        /**
+         * TODO: Change the version number of check as per version in which this change will be merged.
+         */
+        MASTER_TASK_THROTTLED_EXCEPTION(
+            org.opensearch.cluster.service.MasterTaskThrottlingException.class,
+            org.opensearch.cluster.service.MasterTaskThrottlingException::new,
+            161,
+            Version.V_1_3_0
```

Comment:
Forgot to change this, will change it in next update along with other feedback.

@shwetathareja, For main branch, I believe this should be V_2_0_0(which is latest version). Right?